v3/reutlingen.py

```diff
@@ -0,0 +1,122 @@
+
+    def validate(self, input_data: Any, **kwargs) -> list:
+        input_match = re.match(self.PATTERN, input_data)
+        if input_match is None:
+            raise ValidationError(code='invalid_tuple_input', reason='invalid point coordinate tuple input')
+        input_data = [float(input_match.group(1)), float(input_match.group(2))]
+
+        return super().validate(input_data, **kwargs)
+
+
+@validataclass
+class ReutlingenRowInput:
+    uid: int = IntegerValidator(allow_strings=True)
+    type: ReutlingenParkingSiteType = EnumValidator(ReutlingenParkingSiteType)
+    coordinates: list = PointCoordinateTupleValidator(NumericValidator(), min_length=2, max_length=2)
+    capacity: str = Noneable(StringValidator())
+    name: str = StringValidator(max_length=255)
+
+
+class ReutlingenConverter(CsvConverter):
+    reutlingen_row_validator = DataclassValidator(ReutlingenRowInput)
+    proj: pyproj.Proj = pyproj.Proj(proj='utm', zone=32, ellps='WGS84', preserve_units=True)
+
+    source_info = SourceInfo(
+        id='reutlingen',
+        name='Stadt Reutlingen',
+        public_url='https://www.reutlingen.de',
+    )
+
+    header_mapping: dict[str, str] = {'id': 'uid', 'ort': 'name', 'Kapazität': 'capacity', 'GEOM': 'coordinates', 'type': 'type'}
+
+    def handle_csv_string(self, data: StringIO) -> ImportSourceResult:
+        return self.handle_csv(list(csv.reader(data, dialect='unix', delimiter=',')))
+
+    def handle_csv(self, data: list[list]) -> ImportSourceResult:
+        import_source_result = self.generate_import_source_result(
+            static_parking_site_inputs=[],
+            static_parking_site_errors=[],
+        )
+
+        mapping: dict[str, int] = self.get_mapping_by_header(self.header_mapping, data[0])
+
+        # We start at row 2, as the first one is our header
+        for row in data[1:]:
+            input_dict: dict[str, str] = {}
+            for field in self.header_mapping.values():
+                input_dict[field] = row[mapping[field]]
+
+            try:
+                input_data: ReutlingenRowInput = self.reutlingen_row_validator.validate(input_dict)
+            except ValidationError as e:
+                import_source_result.static_parking_site_errors.append(
+                    ImportParkingSiteException(
+                        uid=input_dict.get('uid'),
+                        message=f'validation error for {input_dict}: {e.to_dict()}',
+                    ),
+                )
+                continue
+
+            parking_site_input = StaticParkingSiteInput(
+                uid=input_data.uid,
+                name=input_data.name,
+                lat=input_data.coordinates[1],
+                lon=input_data.coordinates[0],
+                type=input_data.type.to_parking_site_type_input(),
+                capacity=0 if input_data.capacity == '' else int(input_data.capacity),
+                static_data_updated_at=datetime.now(tz=timezone.utc),
```

Comment:
As the `ort` is always a street name, it would be nice for the user to use this as an address, too. If you do that, please add the city to the string.

Reply:
Good hint! I added an ```address``` field, do you think it is better adding the field without Postcode?

Reply:
I guess it makes sense without a postalcode, it will still work as a navigation system input.